frontend/src/app/components/action/Actions.css

```diff
@@ -20,6 +21,7 @@
   background-color: var(--surface-background-white) !important;
   box-shadow: var(--surface-shadow-medium);
   border: none !important;
+  width: 272px;
```

Comment:
Any option instead of setting specific pixels ?

Reply:
as per design it is required to have specific width however I have changed it to rem size for better responsiveness